src/kola-junit

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env python3
+
+'''
+    This script is used in Jenkins to convert kola's `report.json` to JUnit
+    XML, which Jenkins knows how to display natively.
+'''
+
+import argparse
+import json
+import os
+import re
+import sys
+
+from junit_xml import TestSuite, TestCase
+
+
+parser = argparse.ArgumentParser()
+parser.add_argument("classname")
+parser.add_argument("koladir")
+parser.add_argument("outputxml")
```

Comment:
Looking at https://github.com/coreos/coreos-ci-lib/pull/156 I think it would be easier to follow if these were `--` parameters: `--classname`, `--koladir`,...